package.json

```diff
@@ -10,6 +10,8 @@
     "cypress:storybook": "cypress run --project ./.storybook/.",
     "dev": "razzle start",
     "lighthouse": "lighthouse http://localhost:7080/ --output-path=./reports/report.html --view",
+    "precommit": "npm run test:lint",
```

Comment:
Inlining the hooks in `scripts` is an older format of adding hooks with husky, we should use the latest format. See https://github.com/typicode/husky#upgrading-from-014

Reply:
I did this and it didn't work, upon further inspection (see top of docs https://github.com/typicode/husky#upgrading-from-014 )these are the docs for the non-stable version of husky.